domain/consensus/processes/blockvalidator/proof_of_work_test.go

```diff
@@ -0,0 +1,81 @@
+package blockvalidator_test
+
+import (
+	"github.com/kaspanet/kaspad/domain/consensus/model/pow"
+	"github.com/kaspanet/kaspad/domain/consensus/ruleerrors"
+	"github.com/kaspanet/kaspad/util"
+
+	"math/rand"
+	"testing"
+
+	"github.com/kaspanet/kaspad/domain/consensus"
+	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/testutils"
+	"github.com/kaspanet/kaspad/domain/dagconfig"
+	"github.com/pkg/errors"
+)
+
+// Set the flag "skip pow" to be false (second argument in the function)
```

Comment:
If you comment here, please explain what the test is trying to check, instead of how. (But you can also add additional comment for the how)

Reply:
fixed :)